models/backbone/vgg.py

```diff
@@ -0,0 +1,172 @@
+import torch
+import logging
+import torch.nn as nn
+
+from ..utils import conv3x3_group, norm_layer, layers, \
+     kaiming_init, constant_init, load_checkpoint
+
+
+def _make_res_layer(block,
+                    inplanes,
+                    planes,
+                    blocks,
+                    stride=1,
+                    dilation=1,
+                    use_gn=False):
+    layers = []
```

Comment:
This function should have a `maxpool` layer for each vgg layer, and add parameter `maxpool=True` to indicate whether to use `maxpool` or not